maestro-client/src/main/java/maestro/Maestro.kt

```diff
@@ -160,8 +163,10 @@ class Maestro(
     }
 
     fun swipeFromCenter(swipeDirection: SwipeDirection, durationMs: Long) {
+        val deviceInfo = deviceInfo()
```

Comment:
this doesnt seem to use the cached info - is that the intent?

Reply:
Yes.

If we use the cached version, then swipe will completely break if the device is rotated mid-flow (or when running `maestro test` in `--continuous` mode).